pkg/cmd/kafka/create/create.go

```diff
@@ -219,6 +226,13 @@ func runCreate(opts *options) error {
 			CloudProvider: &opts.provider,
 		}
 
+		if opts.marketplaceId != "" && opts.marketplaceType != "" {
```

Comment:
It looks like we always need to require and send pair of marketplace type and marketplace id